### PR TITLE
techcrunch.com

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/114932

<details><summary>Screenshot:</summary>

Снимок экрана 2022-04-08 в 16 56 31

</details><br/>

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -121,6 +121,7 @@ theoryandpractice.ru##a[href^="https://form.typeform.com/"]
 !######### Subscription popups add to subscriptions.txt #########
 !
 techacademy.jp##.pum-overlay[data-popmake*="640px"]
+techcrunch.com##.announcement-banner[style="position: fixed;"]
```
Comment (zloyden) [inline AnnoyancesFilter/sections/popups.txt:124]: Is that `[style="position: fixed;"]` needed?

**Comment by Alex-302** [comment]
Also comments are broken. Better fix them
image

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (jellizaveta) [reply AnnoyancesFilter/sections/popups.txt:124]: This is part of the content and therefore in my opinion can be on the page

Comment (Alex-302) [reply AnnoyancesFilter/sections/popups.txt:124]: Got it. Already found fixed banner.
<details><summary>Screenshot:</summary>

image

</details>

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**